### DISCUSSION TITLE
Paodns+way组合油管经常出现无法播放情况

### DISCUSSION THREAD

**Discussion by Makerlong** [kkkgo/PaoPaoGateWay#40] [问和答] [answered]
目前两天经常出现这种情况，时不时就会如下图所示，只有重启网关才会正常。请教大佬是哪里设置问题么。
![image](https://github.com/kkkgo/PaoPaoGateWay/assets/129925998/4ceb4483-b953-436c-9a20-ad37311c68be)

**Comment by Lovest20018** [comment, marked as answer]
关了quic试试？你的节点支持udp吗？

**Reply by Makerlong** [reply]
好的，我测试一下

**Comment by kkkgo** [comment]
可以配置udp_enable为no或者定制镜像SNIFF=yes试试。
